devtools/startup/DevToolsStartup.jsm

```diff
@@ -1474,20 +1474,14 @@ function saveRecordingInDB(description) {
 
 async function saveRecordingUser(user) {
   if (!user) {
+    // TODO maybe change this pref name
```

Comment:
Still on the fence about this. It'll probably work, but only by chance because the old payload didn't have `.sub`. I have no idea if anything else reads this payload for other fields.